src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -519,16 +537,42 @@ private void sendCitizenPackets(Set<EntityPlayerMP> oldSubscribers, boolean hasN
                 if (citizen.isDirty() || hasNewSubscribers)
                 {
                     subscribers.stream()
-                               .filter(player -> citizen.isDirty() || !oldSubscribers.contains(player))
-                               .forEach(player -> MineColonies.getNetwork().sendTo(new ColonyViewCitizenViewMessage(this, citizen), player));
+                            .filter(player -> citizen.isDirty() || !oldSubscribers.contains(player))
+                            .forEach(player -> MineColonies.getNetwork().sendTo(new ColonyViewCitizenViewMessage(this, citizen), player));
                 }
             }
         }
     }
 
+    /**
+     * Sends packages to update the workOrders.
+     *
+     * @param oldSubscribers    the existing subscribers.
+     * @param hasNewSubscribers the new subscribers.
+     */
+    private void sendWorkOrderPackets(Set<EntityPlayerMP> oldSubscribers, boolean hasNewSubscribers)
+    {
+        if (getWorkManager().isDirty() || hasNewSubscribers)
+        {
+            for (AbstractWorkOrder workOrder : getWorkManager().getWorkOrders().values())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'workOrder' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)